crates/modelardb_server/src/context.rs

```diff
@@ -324,11 +324,20 @@ impl Context {
     ) -> Result<Option<Arc<ModelTableMetadata>>, ModelarDbError> {
         let database_schema = self.default_database_schema()?;
 
-        let table = database_schema.table(table_name).await.ok_or_else(|| {
-            ModelarDbError::DataRetrievalError(format!(
-                "Table with name '{table_name}' does not exist."
-            ))
-        })?;
+        let table = database_schema
+            .table(table_name)
+            .await
+            .map_err(|e| {
+                ModelarDbError::DataRetrievalError(format!(
+                    "Failed to retrieve metadata for '{table_name}' due to: {}",
```

Comment:
Also very minor but it seems that we use string interpolation with most other `map_err()` calls. Maybe change to `{error}`.

Reply:
Fixed.